.NET/Microsoft.Recognizers.Definitions/French/DateTimeDefinitions.cs

```diff
@@ -631,5 +634,9 @@ public static class DateTimeDefinitions
 			{ "newyeareve", new string[] { "réveillondenouvelan", "reveillondenouvelan", "lasaint-sylvestre", "lasaintsylvestre" } }
 		};
 		public const string NightRegex = @"\b(minuit|nuit)\b";
+		public static readonly Dictionary<string, int> WrittenDecades = new Dictionary<string, int>
+		{
+			{ "xxx", 20 }
```

Comment:
The dictionaries for the languages not yet supported could have been left empty, no?

Reply:
I didn't find a way to left the dictionary empty in yaml.